src/aspire/source/relion.py

```diff
@@ -90,10 +106,10 @@ def __init__(
         # Get the 'mode' (data type) - TODO: There's probably a more direct way to do this.
         mode = int(mrc.header.mode)
         dtypes = {0: "int8", 1: "int16", 2: "float32", 6: "uint16"}
-        ensure(
-            mode in dtypes,
-            f"Only modes={list(dtypes.keys())} in MRC files are supported for now.",
-        )
+        assert (
```

Comment:
Just curious about this. Why switch from `ensure` to `assert`? Are we phasing out `ensure` across the entire codebase?

Reply:
My impression was that we are phasing out `ensure` but maybe I misremembered? @garrettwrong 

That could go in the standards/conventions doc if so

Reply:
Cool, made an issue and a note on #569